client/src/components/SideBar/Channels/index.tsx

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useSelectedGroup } from '../../../hooks/useSelectedGroup';
+import { socket } from '../../../util/socket';
 import ChannelListItem from './ChannelListItem';
+import MeetingUserList from './MeetingUserList';
 import { ChannelWrapper, ChannelType } from './style';
 
 interface Props {
   channelType: 'chatting' | 'meeting';
 }
 
+interface IMeetingUser {
+  [key: number]: object[];
+}
+
 function Channels({ channelType }: Props) {
   const selectedGroup = useSelectedGroup();
   const channels =
     selectedGroup?.[channelType === 'chatting' ? 'chattingChannels' : 'meetingChannels'];
+  const [meetingUser, setMeetingUser] = useState<IMeetingUser>({});
+
+  useEffect(() => {
+    const meetingchannelList = selectedGroup.meetingChannels.map(
+      (channel: { id: number }) => channel.id,
+    );
+
+    socket.emit('MeetingChannelList', selectedGroup.code, meetingchannelList);
+  }, [selectedGroup]);
+
+  useEffect(() => {
+    socket.on('MeetingUserList', (meetingUserList) => {
+      setMeetingUser({ ...meetingUserList });
+    });
```

Comment:
```suggestion
  useEffect(() => {
    socket.on('MeetingUserList', (meetingUserList) => {
      setMeetingUser({ ...meetingUserList });
    });
    
    return () => {
      socket.off('MeetingUserList');
    }
```
클린업! 클린업! 클린업!
그리고 이벤트 이름 enum이나 상수로 처리하면 좋을듯!